__tests__/app.test.js

```diff
@@ -106,6 +106,9 @@ describe("/api/articles", () => {
         .expect(200)
         .then(({ body: { articles } }) => {
             expect(articles).toBeInstanceOf(Array);
+            if (articles.length === 0){
+                return;
```

Comment:
You can do a more specific check here such as `toHaveLength` as currently your check with the return means that if it was length 0 you would just skip the rest of the test block.